alexandria/core/tests/test_views.py

```diff
@@ -398,6 +399,70 @@ def test_move_document_to_new_category(
     assert response.status_code == HTTP_200_OK
 
 
+@pytest.mark.parametrize(
+    "to_category,expected_status",
+    [
+        ("same", HTTP_201_CREATED),
+        ("new", HTTP_201_CREATED),
+        ("not_defined", HTTP_201_CREATED),
+        ("non_existent", HTTP_400_BAD_REQUEST),
+        ("not_allowed", HTTP_400_BAD_REQUEST),
+    ],
+)
+def test_copy_document(
+    admin_client,
+    category_factory,
+    file_factory,
+    document_factory,
+    to_category,
+    expected_status,
+):
+    category_not_allowed = category_factory.create(allowed_mime_types=["plain/text"])
+    category = category_factory()
+    document = document_factory(category=category)
+    file_factory.create(document=document, name="Image.jpeg", mime_type="image/jpeg")
+
+    if to_category == "non_existent":
+        target_category_pk = 999
```

Comment:
This may cause flaky test if for some reason a category with ID 999 will eventually exist.

For such things, it's better to call the `category_factory()`, take it's primary key, then delete it before commencing with the test. Alternatively, just `assert` that it's missing, with a comment explaining why

Reply:
I like the approach of creating and deleting a category for the test, changed it now in this way